core/src/main/java/org/restcomm/media/core/configuration/MgcpControllerConfiguration.java

```diff
@@ -67,6 +70,17 @@ public void setPort(int port) {
         }
         this.port = port;
     }
+
+    public int getMgcpBufferSize() {
+        return mgcpBufferSize;
+    }
+
+    public void setMgcpBufferSize(int mgcpBufferSize) {
+        if (mgcpBufferSize < 2000 || mgcpBufferSize > 10000) {
```

Comment:
what was the criteria for these limits?

Reply:
I don't have any specific reason for limit. Do you have an idea about the limit, if not let me study that in detail.

Reply:
just make sure it's a positive number, I guess it's up to user to decide.